ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+1. PropTypes are used for Typechecking. Typechecking is a way to catch erorrs and rule out errors that you can possibly get. This is important because as
+an application scales more bugs have the chance of being introduced and Typechecking can reduce the amount of possible errors.
+
+2. When developing in React every Component follows a cycle of Mounting, Updating, and Unmounting this is called the Component life-cycle. During this cycle there are many methods available for you to use during the phases. componentDidMount is a useful life-cycle method that is called during the Mouting phase of the application. componentDidMount is only called once in this life-cycle and is typically used for setting state using data from a API call.
+
```

Comment:
Yes that's true. componentDidMount is used in API calls but not exclusively in API calls. It's used to set pretty much any data state. An example would be the dummy data from your Instagram project earlier this week.